src/get-own-property-symbols.js

```diff
@@ -258,9 +258,9 @@
           descriptor.value = function () {
             var
               str = toString.call(this),
-              tst = this[Symbol.toStringTag]
+              tst = (typeof this !== "undefined" && this !== null) ? this[Symbol.toStringTag] : this
```

Comment:
why use the more verbose form, when `this != null` is more concise and just as bulletproof?

Reply:
I think the != null to catch the undefined too is less readable than two separate strict equality check, and more error prone when the code needs to be updated. It usually requires a comment explaining that you need to catch nulls and undefined but not falsey values here.
The minifier would eventually produce a more efficient code at the end in both cases.
However, as you prefer.

Reply:
I don’t think a comment is necessary; use of == null is the comment.